index.html

```diff
@@ -180,5 +180,44 @@ <h1>About me</h1>
                 </ul>
             </div>
         </section>
+
+        <section class="contact-section">
+           <p class="contact-par">I'm always interested in hearing about new projects, so if you'd like to chat get in touch</p>
+            <form>
+                <ul class="first-item-inner-about">
+                    <li>
+                        <li>
+                            <label for="fullname"></label>
+                            <input id="fullname" type="text" name="full_name" placeholder="fullname" size="30" required>
+                        </li>
+                        <li>
+                            <label for="email"></label>
+                            <input id="email" type="email" name="email_name" placeholder="email" size="25" required>
+                        </li>
+                        <li>
+                            <label for="textarea"></label>
+                            <textarea id="textarea" type="textarea" name="textarea" size="500" required>Enter your comment here...</textarea>
+                
+                        </li>
+                       
+                    </li>
```

Comment:
-  Kindly make sure you implement the functionality I list below for the best functionality of the contact form  :+1:  :grinning: :grinning::grinning::grinning:

- All input fields have the correct associated placeholder text. Kindly add the placeholder to your text area instead of that pres write line `Enter your comment here...` you can even put it in place holder `placeholder=" Enter your comment here..."` :grinning: :grinning: 

 - Form uses the `action` and `method` attribute to integrate with [formspree service](https://formspree.io/html). :grinning: :grinning: :grinning: 
   * `action` value should be similar to `https://formspree.io/f/{form_id}`
   * `method` value equals `POST` 
  
## Example of the code you are supposed to add in your form 

  `<form action="https://formspree.io/f/mvoyalje" method="post">`


-  Please check the `network` tab in your browser dev tools to confirm that the form is submitted successfully with valid input values. :grinning: :grinning: :grinning: :grinning:

Reply:
Fixed.